src/main/target/common_fc_pre.h

```diff
@@ -41,6 +41,7 @@
 #ifdef STM32F3
 #define MINIMAL_CLI
 #define USE_DSHOT
+#define USE_GYRO_DATA_ANALYSE
```

Comment:
Any reason this is not added to F4 / F7?

Also, this can not be enabled for all F3s, since it blows out flash space (at least) for OMNIBUS.